django_x509/base/models.py

```diff
@@ -310,6 +315,7 @@ def _verify_extension_format(self):
             if not ('name' in ext and 'critical' in ext and 'value' in ext):
                 raise ValidationError(msg)
 
+
```

Comment:
not sure this new line is necessary, always keep your changes to the least minimum required to complete a task